src/coreclr/jit/block.h

```diff
@@ -1291,6 +1254,47 @@ struct BasicBlockList
     }
 };
 
+// BBswtDesc -- descriptor for a switch block
```

Comment:
You inserted this in an odd location, because there is a huge comment above `BasicBlockList` that covers `BasicBlockList` as well as `flowList`; why not put this before that comment

Reply:
Moved.